app/models/ext_management_system.rb

```diff
@@ -193,6 +193,9 @@ def self.create_from_params(params, endpoints, authentications)
   supports_not :swift_service
   supports_not :vm_import
   supports_not :volume_availability_zones
+  supports_not :add_storage
+  supports_not :add_host_initiator
+  supports_not :add_volume_mapping
```

Comment:
Can you move these up so they are ordered alphabetically

Reply:
Alphabetized